_config/config.yml

```diff
@@ -13,10 +13,10 @@ SilverStripe\Core\Injector\Injector:
   SilverStripe\Forager\Interfaces\IndexingInterface:
     class: SilverStripe\ForagerBifrost\Service\BifrostService
     constructor:
-      client: '%$Elastic\EnterpriseSearch\Client'
+      client: '%$Elastic\EnterpriseSearch\Client.managementClient'
```

Comment:
should this reference this module - eg bifrostManagementClient ? Just thinking its a fairly generic name - or is this relying on 'after' to override the enterprise search one?

Reply:
I just remembered the enterprise search module has the env-var condition so won't be creating a client anyway so this should be all good